.github/workflows/trigger-dependent-repo-update.yml

```diff
@@ -0,0 +1,26 @@
+name: Trigger Dependent Repo Update Workflow
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [ master ]
+
+jobs:
+  trigger:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Generate authentication token
+        id: gen-token
+        uses: actions/create-github-app-token@v1
+        with:
+          app-id: ${{ secrets.CVAT_BOT_APP_ID }}
+          private-key: ${{ secrets.CVAT_BOT_PRIVATE_KEY }}
+          repositories: ${{ secrets.CVAT_DEPENDENT_REPO }}
+
+      - name: Trigger repository dispatch
```

Comment:
Have you considered using a `workflow_dispatch` event instead? It seems like it would be more useful, since you could then trigger the dependent workflow manually through the UI. It also doesn't require a 3rd-party action, since you can just use `gh workflow run` on the command line.